.github/workflows/pytest.yml

```diff
@@ -34,3 +34,15 @@ jobs:
           pytest --cov=./ --cov-report=xml -n auto
       - name: Codecov
         uses: codecov/codecov-action@v3.1.0
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.x'
+      - name: Install dependencies
+        run: pip install build
+      - name: Build package
+        run: python -m build
```

Comment:
Shouldn't this block (this last line in particular) replace the `pip install -e ".[tests]"` step?

And also update `cache-dependency-path` for `pyproject.toml`?

Reply:
No, this is just to test that building the package works in general, it's not supposed to test it.